include/core/ib_particle.h

```diff
@@ -116,8 +116,12 @@ class IBParticle
   std::vector<Point<dim>> previous_positions;
   // The fluid force applied on the particle.
   Tensor<1, 3> fluid_forces;
+  Tensor<1, 3> fluid_viscous_forces;
+  Tensor<1, 3> fluid_pressure_forces;
   // The fluid force applied on the particle at the end of the last time step.
   Tensor<1, 3> previous_fluid_forces;
+  Tensor<1, 3> previous_fluid_viscous_forces;
+  Tensor<1, 3> previous_fluid_pressure_forces;
```

Comment:
this is not necessary. the previous value of the force are only use for initial guess when we solved implicitly the DEM

Reply:
I guess this way the checkpoint is coherent, that shouldnt be an issue.